ror-tools-core/src/main/scala/tech/beshu/ror/tools/core/patches/internal/filePatchers/JarManifestModifier.scala

```diff
@@ -67,7 +67,9 @@ object JarManifestModifier {
     originalJarFile.entries().asIterator().asScala.foreach { entry =>
       val name = entry.getName
       if (!name.equalsIgnoreCase("META-INF/MANIFEST.MF")) {
-        jarOutput.putNextEntry(entry)
+        val newEntry = new JarEntry(name)
```

Comment:
I refactored it a bit by creating new JarEntry, maybe it was causing problems. But I'm not sure it was the underlying issue. It is very hard to replicate, as it fails only sometimes and onyl for newest ES versions.